testsuite/cluster-test/src/experiments/performance_benchmark.rs

```diff
@@ -275,6 +277,19 @@ impl PerformanceBenchmark {
             .report
             .report_txn_stats(self.to_string(), stats, window);
 
+        // Backup throughput
+        if self.backup {
+            let bytes_per_sec = pv.avg_backup_bytes_per_second().unwrap_or(0.0);
```

Comment:
Do you mind adding here smth like `.map_err(|e|warn!("Failed to get avg_backup_bytes_per_second: {}", e)` so that if something happens we know what the error was?

Reply:
@andll it's done inside of "avg_backup_bytes_per_second".

Reply:
Oh, cool. I overlooked it